src/Home/Header.tsx

```diff
@@ -14,6 +20,32 @@ class Header extends React.Component<{}, {}> {
           Not only do you get to enjoy free high quality games everywhere,
           but you also can study how they are made, change them, and contribute back to the community!
         </Typography>
+        _links();
```

Comment:
_links() is NOT being interpreted as code. This is being interpreted as text. Thats why coverage decreased. You need to use {this._links()} instead.

Reply:
Yep -- I already fixed this in my local copy. I didn't push yet because I got sidetracked by our server problem. I'll work on it tomorrow (well.. it's 3:15 am, so I guess today).